lib/modules/platform/github/index.ts

```diff
@@ -277,7 +277,10 @@ export async function initRepo({
     if (!repo.defaultBranchRef?.name) {
       throw new Error(REPOSITORY_EMPTY);
     }
-    if (repo.nameWithOwner && repo.nameWithOwner !== repository) {
+    if (
+      repo.nameWithOwner &&
+      repo.nameWithOwner.toLocaleLowerCase() !== repository.toLocaleLowerCase()
```

Comment:
I think uppercase is more safe, as there are some chars which don't have a lowercase representation.